### ISSUE TITLE
Failed to start saltcorn as a service

### ISSUE THREAD

**Issue by Txikivasco** [saltcorn/saltcorn#1473]
Hello,

I've made a self-hosted installation on a linux server and can run saltcorn. Already logged in, etc.

My issue is I can't run it as a service

here's what I get when trying systemctl start saltcorn

![imagen](https://user-images.githubusercontent.com/99097255/202900454-68469332-2bef-4313-baa0-242dd3eb3f5d.png)

**Comment by AleksandrSokolov** [comment]
Hi @Txikivasco Please don't use reserved or used ip port numbers.
I think In you case port number 82 is bad choice.

Port numbers range from 0 to 65535, but only port numbers 0 to 1023 are reserved for privileged services and designated as well-known ports.

**Comment by Txikivasco** [comment]
Thank you for the advice. I just picked a random port that was free on my server, and kind of similar number to the one in the installation guide (80).

I suggest this recommendation to be written down in the instructions as some users like me may not know that kind of basic things. That would save a lot of trial and error.

Anyway, I managed to fix the issue by changing the ExecStart path to /usr/bin/saltcorn serve. Now it runs smoothly as a service. Tried to reboot the serve twice and service starts at boot, so it's ok for me.
